include/zephyr/sys/util.h

```diff
@@ -685,6 +685,16 @@ char *utf8_trunc(char *utf8_str);
  */
 char *utf8_lcpy(char *dst, const char *src, size_t n);
 
+/**
+ * @brief Checks if the given string @p str is UTF-8 encode.
+ *
+ * @param str Target string
+ * @param len The length of string @p str
+ *
+ * @return true if @p str is UTF-8 encode, and false otherwise.
```

Comment:
```suggestion
 * @return true if @p str is UTF-8 encoded, or false otherwise.
```

Small typo + proposition to use 'or'